code/modules/research/xenobiology/crossbreeding/warping.dm

```diff
@@ -0,0 +1,1135 @@
+		slowed_down = crossing
+		slowed_down.add_movespeed_modifier(/datum/movespeed_modifier/status_effect/sepia_rune)
+		addtimer(CALLBACK(src, .proc/normal_speed, slowed_down), 50)
+		if(slowed_down.age < 50) //not counterable by just being ultra young
+			slowed_down.age = 50
+
+		slowed_down.age += 5
+		if(slowed_down.age < 100)
+			to_chat(slowed_down, "<span class='warning'>The rune is absorbing your life force! You feel older!</span>")
+
+		if(slowed_down.age >= 100)
+			to_chat(slowed_down, "<span class='warning'>You feel incredibly old! You should avoid this rune at all cost!</span>")
+			slowed_down.become_nearsighted(type)
+			if(slowed_down.gender == MALE)
+				slowed_down.facial_hairstyle = "Beard (Very Long)"
+				slowed_down.update_hair()
+
+		if(slowed_down.age >= 120)
+			slowed_down.visible_message("<span class='notice'>[slowed_down] is too old and weak to resist the rune anymore!</span>")
+			slowed_down.dust(0,1,0)
+
+
+obj/effect/warped_rune/sepiaspace/proc/normal_speed(mob/living/slowed_down)
+	slowed_down.remove_movespeed_modifier(/datum/movespeed_modifier/status_effect/sepia_rune)
+
+/*Cerulean crossbreed : creates a hologram of the last person that stepped on the tile */
+
+
+/obj/item/slimecross/warping/cerulean
+	colour = "cerulean"
+	runepath = /obj/effect/warped_rune/ceruleanspace
+	effect_desc = "Draws a rune creating a hologram of the last living thing that stepped on the tile."
+	max_charge = 1
+	warp_charge = 1
+
+
+/obj/effect/warped_rune/ceruleanspace
+	desc = "A shadow of what once passed these halls, a memory perhaps?"
+	icon_state = "holo_rune"
+	max_cooldown = 100
+	///hologram that will be spawned by the rune
+	var/obj/effect/overlay/holotile
+	///mob the hologram will copy
+	var/mob/living/holo_host
+	///used to remember the recent speech of the holo_host
+	var/list/recent_speech = list()
+
+
+/obj/effect/warped_rune/ceruleanspace/Initialize()
+	. = ..()
+	START_PROCESSING(SSprocessing, src)
+
+
+/obj/effect/warped_rune/ceruleanspace/process()
+	if(cooldown > world.time)
+		return
+	cooldown = max_cooldown + world.time
+	if(holotile && length(recent_speech))
+		holotile.say(recent_speech[pick(recent_speech)]) //say one of the 5 latest sentence said by the holo_host
+
+
+///makes a hologram of the mob stepping on the tile, any new person stepping in will replace it with a new hologram
+/obj/effect/warped_rune/ceruleanspace/Crossed(atom/movable/crossing)
+	. = ..()
+	if(!istype(crossing,/mob/living))
+		return
+
+	if(locate(holotile) in rune_turf)//here to delete the previous hologram,
+		qdel(holotile)
+
+	holo_host = crossing
+	holotile = new(rune_turf) //setting up the hologram to look like the person that just stepped in
+	holotile.icon = holo_host.icon
+	holotile.icon_state = holo_host.icon_state
+	holotile.alpha = 100
+	holotile.name = "[holo_host.name] (Hologram)"
+	holotile.add_atom_colour("#77abff", FIXED_COLOUR_PRIORITY)
+	holotile.copy_overlays(holo_host, TRUE)
+
```

Comment:
make this reset the cooldown as well, both to stop lag-spamming by putting a shitton of monkeys on a conveyorbelt or somethign and to make it less apparent to the person who just walked across the rune that the rune is gonna be repeating their speech once every 10 seconds